doc/sphinx/source/quickstart/running.rst

```diff
@@ -4,138 +4,88 @@
 =================================
 
-See Section :doc:`Recipes <../recipes/index>` for a description of all
-available recipes.
-
-To see a list of installed recipes run
-
-.. code:: bash
+Although ESMValTool can be used just to simplify the management of data
+and the creation of your own analysis code, one of its main strengths is the
+continuously growing set of diagnostics and metrics that it directly provides to
+the user. These metrics and diagnostics are provided as a set of preconfigured
+recipes that users can run or customize for their own analysis.
+The latest list of available recipes can be found :ref:`here <esmvaltool:recipes>`.
 
-	esmvaltool recipes list
+In order to make the management of these installed recipes easier, ESMValTool
+provides the ``recipes`` command group with utilities that help the users in
+discovering and customizing the provided recipes.
 
-Running multiple recipes
-========================
+The first command in this group allows users to get the complete list of installed
+recipes printed to the console:
 
-It is possible to run more than one recipe in one go: currently this relies on the user
-having access to a HPC that has ``rose`` and ``cylc`` installed since the procedure involves
-installing and submitting a Rose suite. the utility that allows you to do this is
-``esmvaltool/utils/rose-cylc/esmvt_rose_wrapper.py``.
+.. code:: bash
 
-Base suite:
------------
-The base suite to run esmvaltool via rose-cylc is `u-bd684`; you can find
-this suite in the Met Office Rose repository at:
+    esmvaltool recipes list
 
-https://code.metoffice.gov.uk/svn/roses-u/b/d/6/8/4/trunk/
+If the user then wants to explore any one of this recipes, they can be printed
+using the following command
 
-When ``rose`` will be working with python3.x, this location will become
-default and the pipeline will aceess it independently of user, unless, of
-course the user will specify ``-s $SUITE_LOCATION``; until then the user needs
-to grab a copy of it in ``$HOME`` or specify the default location via ``-s`` option.
+.. code:: bash
 
-Environment:
-------------
-We will move to a unified and centrally-installed esmvaltool environment;
-until then, the user will have to alter the env_setup script:
+    esmvaltool recipes show recipe_name.yml
 
-``u-bd684/app/esmvaltool/env_setup``
+And finally, to get a local copy that can then be customized and run, users can
+use the following command
 
-with the correct pointers to esmvaltool installation, if desired.
+.. code:: bash
 
-To be able to submit to cylc, you need to have the `/metomi/` suite in path
-AND use a `python2.7` environment. Use the Jasmin-example below for guidance.
+    esmvaltool recipes get recipe_name.yml
 
-Jasmin-example:
----------------
-This shows how to interact with rose-cylc and run esmvaltool under cylc
-using this script:
+Note that the ``esmvaltool run recipe_example.yml`` command will first look if
+``recipe_example.yml`` is the path to an existing file.
```

Comment:
we don't actually have a `recipe_example.yml` anymore, so maybe use the name of an existing recipe?

Reply:
Which recipe would you suggest?

Reply:
`recipe_python.yml` maybe? If that one's not there or it fails, then it means King Kong finally smashed the Empire State Building :grin:

Reply:
The downside of that recipe is that it lives in the examples directory, so you would need to use `examples/recipe_python.yml` everywhere, making the examples more complicated.

Reply:
I added a note that there is no recipe with that name and they need to use one that exists

Reply:
OK if you think that's more user-friendly than running an existing one even tho in `examples` - maybe we should do a recipe example in `recipes`? - am impartial, either thing works for me